Source/Core/Core/PrimeHack/PrimeUtils.h

```diff
@@ -41,6 +43,17 @@ void set_beam_owned(int index, bool owned);
 void set_visor_owned(int index, bool owned);
 void set_cursor_pos(float x, float y);
 
+
+std::string as_hex_string(u32 val);
+void print_to_log(const char* message, const char* file, int line, Common::Log::LOG_LEVELS level, Common::Log::LOG_TYPE type);
+void print_to_log(const char* message, const char* file, int line);
+void print_to_log(const char* message);
+
+void print_to_log(std::string message, const char* file, int line, Common::Log::LOG_LEVELS level, Common::Log::LOG_TYPE type);
+void print_to_log(std::string message, const char* file, int line);
+void print_to_log(std::string message);
```

Comment:
The amount of added functions here is obsessive. Please reduce this to 1 or 2. I recommend one function taking in a format and arguments. Think about the use cases and what is most commonly needed.